lib/hammer_cli_foreman/compute_resource.rb

```diff
@@ -70,6 +70,11 @@ def print_data(data)
 
     class CreateCommand < HammerCLIForeman::CreateCommand
 
+
+      option "--public_key_path", "PUBLIC_KEY_PATH", _("Path to a file that contains oVirt public key (For oVirt only)"), :attribute_name => :option_public_key_path,
```

Comment:
If you use `:attribute_name => :option_public_key` you can remove `request_params` definition completely.

Reply:
I tried it, but it didn't work - the public_key was not in params['compute_resource']